runtime/common/src/parachains.rs

```diff
@@ -142,6 +263,44 @@ pub enum Origin {
 	Parachain(ParaId),
 }
 
+/// An offence that is filed if the validator has submitted a double vote.
+#[derive(RuntimeDebug)]
+#[cfg_attr(feature = "std", derive(Clone, PartialEq, Eq))]
+pub struct DoubleVoteOffence<Offender> {
+	/// The current session index in which we report a validator.
+	session_index: SessionIndex,
+	/// The size of the validator set in current session/era.
+	validator_set_count: u32,
+	/// An offender that has submitted two conflicting votes.
+	offender: Offender,
+}
+
+impl<Offender: Clone> Offence<Offender> for DoubleVoteOffence<Offender> {
+	const ID: Kind = *b"double-vote:doub";
```

Comment:
```suggestion
	const ID: Kind = *b"parachain:double";
```

To differentiate from other kinds of double vote.

Reply:
```suggestion
	const ID: Kind = *b"para:double-vote";
```
alternatively